signal/micro/kernels/framer_test.cc

```diff
@@ -160,6 +160,7 @@ void TestFramerReset(int* input_dims_data, int16_t* input_data,
 }
 
 }  // namespace
+}  // namespace tflite
```

Comment:
Do `TF_LITE_MICRO_TEST`s need to be in the global namespace? Can this just go all the way at the end of the file?

Reply:
TF_LITE_MICRO_TEST is just a macro for main(){} and all of our test are formatted like that  currently.